runtime/near-test-contracts/src/main.rs

```diff
@@ -0,0 +1,30 @@
+use std::io::Write as _;
```

Comment:
I'm a bit surprised there aren't any changes to cargo files.

Reply:
Cargo implicitly considers presence of `src/main.rs` to add a `[bin]` target, just like presence of `src/lib.rs` implicitly adds a `[lib]` target and `tests/*` each add a new `[test]`.